src/board-service/reducer.ts

```diff
@@ -0,0 +1,17 @@
+import { Reducer } from "redux";
+
+interface BoardState {
+   score: number,
+}
+
+const inititalState ={
+   score:0,
+} as BoardState
+
+export const boardReducer: Reducer<BoardState> = (state: BoardState | undefined, action: any):any => {
```

Comment:
will change later

Reply:
it is old version, now looks like boardReducer = (state: BoardState, action: any): BoardState =>